api/handle_scenario_testrun.go

```diff
@@ -37,3 +38,38 @@ func handleCreateScenarioTestRun(uc usecases.Usecases) func(c *gin.Context) {
 		})
 	}
 }
+
+func handleListScenarioTestRun(uc usecases.Usecases) func(c *gin.Context) {
+	return func(c *gin.Context) {
+		ctx := c.Request.Context()
+		organizationId, err := utils.OrganizationIdFromRequest(c.Request)
+		if presentError(ctx, c, err) {
+			return
+		}
+		scenarioID := c.Query("scenario_id")
+
+		usecase := usecasesWithCreds(ctx, uc).NewScenarioTestRunUseCase()
+		testruns, err := usecase.ListTestRunByScenarioId(ctx, organizationId, scenarioID)
+		if presentError(ctx, c, err) {
+			return
+		}
+		c.JSON(http.StatusOK, pure_utils.Map(testruns, dto.AdaptScenarioTestRunDto))
```

Comment:
just for the sake of following the convention in (most of) the codebase, please also wrap it in a `gin.H{"scenario_test_runs": ...}` to return